enqueue/fs/0.4/manifest.json

```diff
@@ -0,0 +1,5 @@
+{
+    "env": {
+        "ENQUEUE_DSN": "file:/%VAR_DIR%/enqueue"
```

Comment:
Ok nevermind, I tested on my ubuntu distro and `file:///` works, once again I should check before talking